frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -76,6 +80,13 @@ pods:
         goal: FINISHED
         cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool snapshot -p {{TASKCFG_ALL_JMX_PORT}} -t $CASSANDRA_SNAPSHOT_NAME -- $CASSANDRA_KEYSPACES
         resource-set: sidecar-resources
+      upload-aws:
+        goal: FINISHED
+        cmd: >
+                chmod a+x {{S3CLI_VERSION}} && eval "echo \"$(sed 's/\"/\\\"/g' < s3config.json)\"" > config.json
```

Comment:
`eval` would otherwise nuke the quote characters in the JSON content of the file, so they need to be escaped again. I could go either way with having the JSON being part of the command, but lean towards it being easier to read/edit JSON in a separate file.